src/main/java/com/google/plantasy/utils/UserUtils.java

```diff
@@ -32,13 +32,21 @@
 import com.google.appengine.api.datastore.Query.SortDirection;
 import com.google.plantasy.utils.QuizTimingPropertiesUtils;
 import java.lang.IllegalArgumentException;
+import java.util.Comparator;
+import java.util.Collections;
 
 public final class UserUtils {
     public static final String SESSION_ID_COOKIE_NAME = "SessionID";
     public static final int ADDED_POINTS = 20;
     private static final Logger log = Logger.getLogger(UserUtils.class.getName());
  
-   
+    static final Comparator<User> RANK = new Comparator<User>() {
+        @Override
+        public int compare(User a, User b) {
+            return Long.compare(b.getScore(), a.getScore());
+        }
+    };
```

Comment:
Do you have to write this function? Doesn't java already come with a library where you can compare objects?

Reply:
I would add some documentation for this

Reply:
I added some documentation for this function. I think I'm going to need a function to do this because I am sorting by a specific attribute of the object, in a certain order.